dev_tools/notebooks/isolated_notebook_test.py

```diff
@@ -40,9 +40,13 @@
 # Please, always indicate in comments the feature used for easier bookkeeping.
 
 NOTEBOOKS_DEPENDING_ON_UNRELEASED_FEATURES: List[str] = [
-    'docs/noise.ipynb',
+    "docs/noise.ipynb",
     # Named topologies
     'docs/named_topologies.ipynb',
+    # these all depends on cirq.SQRT_ISWAP
```

Comment:
I think this is actually not needed anymore because `cirq.SQRT_ISWAP` and `cirq.SQRT_ISWAP_INV` changes have already been [released](https://github.com/quantumlib/Cirq/releases/tag/v0.12.0). You can remove these and also revert the `pip install --pre` changes. 

This PR has been stalling for a while :)

Reply:
I made the suggested changes, and all the test are passing locally. Thank you for helping me out.